internal/cephfs/controllerserver.go

```diff
@@ -243,6 +243,20 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 		}
 	}()
 
+	// check for number of snapshots for volume-volume cloning
+	if pvID != nil && parentVol != nil {
+		// Kernel client only supports up to 400 snapshots per filesystem
+		// see https://tracker.ceph.com/issues/21420
```

Comment:
you explain this 2x now, maybe place the link at the `maximumSnapshotsOnSubVolume` definition?